Yosemite/Yosemite/Model/Payments/CardPresentPaymentsConfiguration.swift

```diff
@@ -124,11 +124,9 @@ private enum Constants {
 }
 
 extension OperatingSystemVersion: Equatable {
-
     public static func == (lhs: OperatingSystemVersion, rhs: OperatingSystemVersion) -> Bool {
-        return lhs.majorVersion == rhs.minorVersion &&
+        return lhs.majorVersion == rhs.majorVersion &&
```

Comment:
Good catch!

Reply:
I can't believe I used to have to write all these by hand. I can't even get a 3-liner right, there must be some mistakes in some of the bigger ones I've written in the past! (All in other projects)